source/installation/kafka-deployments/install-confluent-kafka.txt

```diff
@@ -15,3 +15,188 @@ Install with Confluent Cloud
+
+            For more information, see `Use API Keys to Control Access in Confluent Cloud 
+            <https://docs.confluent.io/cloud/current/access-management/authenticate/api-keys/api-keys.html>`__. 
+
+         .. step:: Configure Kafka cluster API keys
+
+            To access your Kafka cluster, you must configure your Kafka cluster API keys. 
+            Navigate to your :guilabel:`Cluster Overview` and on the left-side navigation 
+            click :guilabel:`API Keys` to create your Kafka cluster API keys.
+
+            Save these configurations to add to your ``user.properties`` file.
+
+            For more information, see `Use API Keys to Control Access in Confluent Cloud 
+            <https://docs.confluent.io/cloud/current/access-management/authenticate/api-keys/api-keys.html>`__.
+
+   .. step:: Gather more configuration information
+
+      The following information is found in the Confluent Cloud console. For more 
+      information see, `Confluent Cloud Documentation 
+      <https://docs.confluent.io/cloud/current/overview.html>`__.
+
+      .. procedure::
+
+         .. step:: Find and save your environment ID 
+
+            From your :guilabel:`Environments` overview page, choose your 
+            cloud environment, and find your environment ID below the environment name 
+            on the right-side panel. 
+
+            The general environment ID form is:
+
+            .. code-block::
+
+               env-ab1cde
+
+         .. step:: Find and save your cluster ID
+
+            Navigate to your :guilabel:`Cluster Overview` page and find your cluster ID 
+            on the right-side information panel. 
+
+            The general cluster ID form is:
+
+            .. code-block::
+
+               abc-1test2
+
+         .. step:: Find and save your cluster's Bootstrap server
+
+            From your :guilabel:`Cluster Overview` page, use the left-side navigation and 
+            click :guilabel:`Cluster Settings`. On the :guilabel:`Cluster Settings` page, 
+            locate your cluster's Bootstrap server.
+
+            The general Bootstrap server form is:
+
+            .. code-block::
+
+               xx.region.provider.confluent.cloud:9092
+
+         .. step:: Find and save your cluster REST endpoint
+
+            On your :guilabel:`Cluster Settings` page, find your cluster's REST 
+            endpoint.
+
+            The general REST endpoint form is:
+
+            .. code-block::
+
+               https://xxx.region.provider.confluent.cloud:443
+
+   .. step:: Add the configuration properties to your ``user.properties`` file
+
+      The location of your ``user.properties`` file varies according to your operating
+      system. For more information, see :ref:`file-location`. 
+
+      Use the following example to add the configuration properties to your 
+      ``user.properties`` file:
+
+      .. code-block::
+
+         spring.profiles.active: confluent-dev
```

Comment:
**Question**

Will this always be `confluent-dev` or simply `confluent`?

```suggestion
         spring.profiles.active: confluent-dev
```

Reply:
It was actually updated to `confluent` so I it has also been updated here now!